.github/workflows/pr.yml

```diff
@@ -29,21 +29,18 @@ jobs:
       - run: |
           gcloud auth configure-docker
 
-      - name: Setup defaults action java and maven properties
-        uses: actions/setup-java@v1
+      - uses: actions/setup-java@v3
+        id: maven
+        name: set java and maven cache
         with:
-          java-version: "17"
+          distribution: 'adopt'
+          java-version: '17'
           architecture: x64
+          cache: 'maven'
           server-id: forgerock-private-releases # Value of the distributionManagement/repository/id field of the pom.xml
           server-username: MAVEN_USERNAME # env variable for username in deploy
           server-password: MAVEN_CENTRAL_TOKEN # env variable for token in deploy
 
-      - name: Cache Maven packages
-        uses: actions/cache@v1
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven2-${{ hashFiles('**/pom.xml') }}
-
```

Comment:
Why remove this? Is it necessary for the release action to work?

Reply:
No the maven cache is provided by `actions/setup-java@v3` 
```yaml
      - uses: actions/setup-java@v3
        id: maven
        name: set java and maven cache
        with:
          distribution: 'adopt'
          java-version: '17'
          architecture: x64
          cache: 'maven'
```
We no need anymore a specific step to create the maven cache.